zanata-client-commands/src/main/java/org/zanata/client/commands/push/PushOptionsImpl.java

```diff
@@ -242,4 +244,17 @@ void setValidate(String validate) {
         this.validate = validate;
     }
 
+    @Override
+    public boolean isMyTrans() {
+        return myTrans;
+    }
+
+    @Option(
+            name = "--my-trans",
+            handler = BooleanValueHandler.class,
+            usage = "Indicates if all uploaded translations will be set to you " +
```

Comment:
I think this may be better "Indicates if all uploaded translations were translated by you"